libraries/Microsoft.Bot.Schema/Teams/BatchOperationState.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Object representing operation state.
+    /// </summary>
+    public class BatchOperationState
```

Comment:
I believe we might be missing a 

`DateTime? RetryAfter`

property

Reply:
Property added. Thanks!